src/org/opensolaris/opengrok/web/WebappListener.java

```diff
@@ -99,9 +101,12 @@ public void contextInitialized(final ServletContextEvent servletContextEvent) {
         }
 
         try {
-            RuntimeEnvironment.getInstance().loadStatistics();
+            RuntimeEnvironment runtimeEnvironment = RuntimeEnvironment.getInstance();
+            if (runtimeEnvironment.getConfiguration().getDataRoot() != null) {
```

Comment:
Do this in load statistics and fix the test where it should test if the file actually exists (provide some nonsense path).